### ISSUE TITLE
WRI Brasil - Critical for launch issues

### ISSUE THREAD

**Issue by monicapatel21** [wri/wri_sites#92]
Wil shortened the team's list of issues to this:

- [x] See if there's a way to update the default Project Narrative Taxonomy to:

```
O projeto [node:title] faz parte do programa de [wri_tokens:topic_and_sub_topic_links_list]. Para informações sobre o projeto, contate <a href="/node/[node:field_primary_contacts:target_id]">[node:field_primary_contacts:entity]</a>. Para solicitar uma entrevista, visite a página de <a href="/imprensa">imprensa</a>.
```

- [x]  Translate "and" in the Author listing to "e"

- [x] Translate Pages to "páginas" (lowercase)

- [x]  See if there's a way to remove padding on the pagination first/last button so it fits in the whole word: https://live-wri-brasil.pantheonsite.io/biblioteca

- [x] Is there an easy way to increase the character length on the GDPR popup text line?

Wil is also planning to work on it so whomever gets here first, please check the boxes when finished or add notes above. :)

**Comment by wthomaswri** [comment]
@aprice42  figured the paginas one, so can skip that one. I'll keep marking the list and updating if I find a solution for some of these.

**Comment by aprice42** [comment]
@wthomaswri I found a fix for this one: `Translate "and" in the Author listing to "e"` but it will require a deployment and then adding the translation configuration for the "and" string in the User interface translations.

**Comment by aprice42** [comment]
@wthomaswri for `See if there's a way to update the default Project Narrative Taxonomy to:` wondering if this is what you are looking for?

![2022-07-05 at 17 15 07](https://user-images.githubusercontent.com/675820/177437015-716be6aa-7e3a-4758-99c0-3e1bcbadab70.png)

I just searched for the string `WRI's [node:title] is part of` to find it. Would changing that translation achieve what you want?

**Comment by aprice42** [comment]
@wthomaswri RE: `Is there an easy way to increase the character length on the GDPR popup text line?`

Unfortunately, it is fixed at 255 characters so you cannot enter text longer than that via the UI. But, it looks like we can work around that by updating the configuration in code to be whatever it needs to be regardless of length, then when we import the config changes it will update to the longer configured value. I was able to do that locally and get something very long as a test.

![2022-07-05 at 17 48 06](https://user-images.githubusercontent.com/675820/177439637-ef1e1468-c739-48be-86f4-d166574b5bc1.png)

This would also require a deployment and you providing me with the translated text that you would like.

**Comment by aprice42** [comment]
@wthomaswri RE `See if there's a way to remove padding on the pagination first/last button so it fits in the whole word`

I am able to get that looking better with this CSS:

```
.pager .pager__items .pager__item--last, .pager .pager__items .pager__item--first {
  width: unset;
  padding: 0 3px
}
```

Maybe that can just be added via asset injection with some of the other overrides you are doing?

![image](https://user-images.githubusercontent.com/675820/177440364-d77ec7e5-8f08-4fbd-90db-d5e361b91296.png)

**Comment by aprice42** [comment]
@wthomaswri ok, I think that covers all the things in the list so far. I need to wrap for the day, and will be back online around 12:00 EST tomorrow. If you want to 

- check out that [PR multi-dev](https://pr-107-wri-brasil.pantheonsite.io/) for the "and" translation and let me know if it looks good.
- and let me know what the GDPR text should change to so I can update the config in code

Then I can get those things deployed for one last review and then pushed to live.

**Comment by wthomaswri** [comment]
Thanks for this @aprice42 

1. Yeah for the Narrative Taxonomy, I tried it through User Interface Translation but it didn't seem to stick. If it's working for you, great..maybe I missed something but let me know.
2. Wasn't able to test the "and" on the PR but I'm assuming it'll work so we can just go ahead.
3. Great on the pagination - I just applied to live.
4. GDPR - This is what they want:

Nosso site utiliza cookies e tecnologias para melhorar sua experiência. Ao navegar por nossos conteúdos, você estará de acordo com o uso das informações que são recolhidas de forma anônima e utilizadas para melhorar o funcionamento das páginas, evitar fraudes e ataques cibernéticos, mensurar a audiência do site e oferecer conteúdos por meio de anúncios personalizados. Para saber mais, conheça nossa [Política de Privacidade](https://wribrasil.org.br/politica-de-privacidade).

Button> "Entendo"
